### ISSUE TITLE
btrfs-assistant

### ISSUE THREAD

**Issue by wvpianoman** [getsolus/packages#4096]
### Please confirm there isn't an existing package request

- [X] I have searched through package requests

### Homepage

https://gitlab.com/btrfs-assistant/btrfs-assistant

### Maintainer

- [ ] Would you like to be the maintainer for this package?

### Why should this be included in the repository?

It is a GUI management tool to make managing a Btrfs filesystem easier

### Are we allowed to redistribute it?

GNU GENERAL PUBLIC LICENSE                        Version 3, 29 June 2007

### What kind of user will use this package, and how many users do you think will use this package?

All users who utilize the btrfs file system

### Link to source archive file

https://gitlab.com/btrfs-assistant/btrfs-assistant/-/archive/2.1.1/btrfs-assistant-2.1.1.tar.gz

**Comment by TraceyC77** [comment]
Also see the other two requests for btrfs manager software. We should probably pick one of the three

**Comment by Julien684** [comment]
No, BTRFS-assistant seems like a good idea to me, as the goal is to be able to manage BTRFS graphically.

**Comment by wvpianoman** [comment]
> > In the end, out of the three, you chose BTRFS-Assistant?
> 
> If there are valid reasons to pick one of the other two rather than this, please provide a reasoned, logical argument for it and we can consider it.

I have only used btrfs-assistant.   Wasn't aware of the other two options.  If either other option is a easier build and integration into Solus, by all means use that one instead .
